pkgs/applications/graphics/loupe/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchgit
+, rustPlatform
+, meson
+, pkg-config
+, cmake
+, gtk4
+, itstool
+, desktop-file-utils
+, ninja
+, gnome-builder
+, wrapGAppsHook4
+, libadwaita
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "loupe";
+  version = "44.alpha";
+
+  src = fetchgit {
+    url = "https://gitlab.gnome.org/BrainBlasted/loupe.git";
+    rev = "aa781d386485d76a647a7b8968b91a08e521c780";
+    sha256 = "sha256-pkHmwnK9lBjpFQbGkEjgTi5rOsYLMK6OydB/k6aMk84=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    hash = "sha256-ay2HNTcYumkwpUZkntJK4vA7puXz5swb8NwC8LG8vW8=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    cmake
+    gtk4
+    itstool
+    desktop-file-utils # update-desktop-database
+    ninja
+    gnome-builder
```

Comment:
This sounds extremely suspicious. Builder is GNOME’s IDE.

Reply:
yeah I just saw it on the README as part of development & I'm not very familiar with gtk software builds

removed